include/eve/detail/spy.hpp

```diff
@@ -74,7 +75,7 @@ namespace spy
   constexpr inline auto ppc_    = detail::arch_info<detail::archs::ppc_>{};
   constexpr inline auto arm_    = detail::arch_info<detail::archs::arm_>{};
 }
-#include <ostream>
+#include <iosfwd>
```

Comment:
Why? This doesn't seem right. You don't have any cpp files, do you?
You streaming operator requires `ostream` included

Reply:
This one escaped the cleaning of SPY. Will fix.